### PR TITLE
[MIG] [17.0] sale_report_hide_price: migrated module

### PR DESCRIPTION
**Pull request by ShwetaSerpentCS**
Responsible: **Amisha**

### CODE REVIEW

**Review by Trivedi-Vacha-SerpentCS — APPROVED** [review]

File: sale_report_hide_price/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Hide Price and Discount in Quotation Report',
+    'author': 'Serpent Consulting Services Pvt. Ltd.',
+    'maintainer': 'Serpent Consulting Services Pvt. Ltd.',
+    'category': 'Sales Management',
+    'website': 'https://www.serpentcs.com',
+    'version': '17.0.1.0.1',
+    'license': 'AGPL-3',
```
Comment (Trivedi-Vacha-SerpentCS) [inline sale_report_hide_price/__manifest__.py]: @ShwetaSerpentCS please check
Odoo 17 Community Edition is licensed under LGPL version 3

**Review by Trivedi-Vacha-SerpentCS — APPROVED** [review]

**Comment by ShwetaSerpentCS** [comment]
@JayVora-SerpentCS 

Please merge this PR.

Thank You.

File: sale_report_hide_price/__manifest__.py (hunk 1 above)
Comment (ShwetaSerpentCS) [reply sale_report_hide_price/__manifest__.py]: > @ShwetaSerpentCS please check Odoo 17 Community Edition is licensed under LGPL version 3

Please review updated code. @Trivedi-Vacha-SerpentCS 

Thank You.

**Comment by ketanserpentcs** [comment]
Testing has been Done with index by Pooja

**Pull request merged**